app/assets/stylesheets/govuk_publishing_components/components/_accordion.scss

```diff
@@ -1,25 +1,231 @@
 @import "govuk/components/accordion/accordion";
 
-.gem-c-accordion {
-  .govuk-accordion__section-heading {
-    // this is a temporary addition to fix the line height when it
-    // is being overridden by styles from govuk-template in collections
-    @include govuk-media-query($until: desktop) {
-      @include govuk-font(24, $weight: bold, $line-height: 1.6);
+$gem-c-accordion-border-width: 3px;
+$gem-c-accordion-bottom-border-width: 2px;
+
+// Buttons within the sections don’t need default styling
+.gem-c-accordion__section-button {
+  @include govuk-font($size: 24, $weight: bold);
+  display: inline-block;
+  margin-bottom: 0;
+  padding-top: govuk-spacing(3);
+}
+
+.gem-c-accordion__section-header {
+  padding-top: govuk-spacing(3);
+  padding-bottom: govuk-spacing(3);
+}
+
+.js-enabled {
+  .gem-c-accordion {
+
+    border-bottom: $gem-c-accordion-bottom-border-width solid $govuk-border-colour;
+  }
+
+  .gem-c-accordion__controls {
+    text-align: left;
+  }
+
+  .gem-c-accordion__open-all {
+
+    position: relative;
+    z-index: 1;
+    border-width: 0;
+    color: $govuk-link-colour;
+    background: none;
+    cursor: pointer;
+    -webkit-appearance: none;
+    margin: 0 0 govuk-spacing(4) 0;
+    padding: 0 0 govuk-spacing(1) 0;
+
+    @include govuk-font($size: 16);
+    @include govuk-link-common;
+    @include govuk-link-style-default;
+
+    // Remove default button focus outline in Firefox
+    &::-moz-focus-inner {
+      padding: 0;
+      border: 0;
     }
   }
-}
 
-.govuk-accordion--condensed {
-  .govuk-accordion__section-button {
-    @include govuk-media-query($from: tablet) {
-      @include govuk-font($size: 19, $weight: bold);
+  // Focus state to change chervon icon to black
+  .gem-c-accordion__open-all:focus {
+    .gem-c-accordion-nav__chevron--stroke {
+      stroke: govuk-colour("black");
```

Comment:
This is a really good idea! I'll implement this into my step by step stuff. Question though, does this work across browsers? I have this vague memory of SVG styling being dodgy on IE.

Reply:
It works across browsers! Great stuff.

Reply:
Thanks, so initially I had used svg background assets & pseudo elements for the icons however I decided to adopt an implementation more inline with step-by-step with the images so I could inline for this reason...

I started to think about various other techniques with inlining svgs within the gem, there might be an existing helper or pattern that exists for this as it's usually _really_ common.  

It would ideally be best to share 1 icon across the accordion + step-by-step and simply use CSS (rotate etc) to reduce the need for assets.  However this the scope of this, it felt isolated per component (also, I felt a way about images within JS)

Reply:
My instinct on this is for now™️ to keep them separate and componentised, then once we've launched this, the step by step and figured out the manuals accordion we can work on a combined solution that suits all our needs. I appreciate that making things good from the start is probably the right thing to do however as the implementation for the wider accordion solution hasn't quite finished baking yet, I'd rather launch stuff than not.

Reply:
Agreed. Let's iterate on these a bit later.